SerhiiBuza/HW5/task2.py

```diff
@@ -0,0 +1,8 @@
+#PrintFibinacci
```

Comment:
good job
here is a more optimized code:

j = int(input("Enter how many Fibonacci numbers you want to see: "))
fibo_list = [0, 1]

while len(fibo_list) < j:
    fibo_list.append(fibo_list[-2] + fibo_list[-1])

print(fibo_list)